api/src/db/migrations/v107_refactor_interstate_transport_essential_question.test.ts

```diff
@@ -0,0 +1,134 @@
+import {
+  generateV106ProfileData,
+  generateV106UserData,
+} from "./v106_add_pet_care_housing_essential_question";
+import { migrate_v106_to_v107 } from "./v107_refactor_interstate_transport_essential_question";
+
+describe("migrate_v106_to_v107", () => {
+  describe("interstateLogistics", () => {
+    it("sets the interStateLogisticsApplicable onboarding question to true for existing users that answered yes", () => {
+      const v106 = generateV106UserData({
+        profileData: generateV106ProfileData({ industryId: "logistics", interstateTransport: true }),
+      });
+      const v107 = migrate_v106_to_v107(v106);
+      expect(v107).toEqual({
+        ...v106,
+        profileData: {
+          ...v106.profileData,
+          isInterstateLogisticsApplicable: true,
+        },
+        version: 107,
+      });
+    });
```

Comment:
given that our code change is adding 2 new fields, I'd expect the test to be testing explicitly for what it sets both fields to in the output. `isInterstateLogisticsApplicable` becomes true for this user, but what should `isInterstateMovingApplicable` be? 

Since the function purpose is to start with one field and determines the value of 2 fields, then in tests, I'd suggest that the tests have assertions that make it clear what we expect from both fields. This comment would hold true for pretty much all the tests here - it'd be nice for them to be explicit about the end-result value of both new fields that are being calculated